app/models/conversion/involuntary/tasks/subleases.rb

```diff
@@ -0,0 +1,18 @@
+class Conversion::Involuntary::Tasks::Subleases < TaskList::OptionalTask
+  attribute :received, :boolean
```

Comment:
These have to be cast for the validation and reloading of the page, else they get assigned as "0" and "1" (which are both true), and we have to do a bunch of extra work.